server/services/store/sqlstore/public_methods.go

```diff
@@ -47,12 +48,12 @@ func (s *SQLStore) AddUpdateCategoryBoard(userID string, categoryID string, boar
 }
 
 func (s *SQLStore) CanSeeUser(seerID string, seenID string) (bool, error) {
```

Comment:
Why not delete this function as its implemented in `mattermostauthlayer`?

Reply:
This required more cleaner, will do this as a part of this ticket: https://mattermost.atlassian.net/browse/MM-59349